Assets/MRTK/SDK/Features/UX/Scripts/Pointers/DefaultPointerMediator.cs

```diff
@@ -142,7 +142,7 @@ public virtual void UpdatePointers()
                         pointer.IsActive = true;
                         unassignedPointers.Remove(pointer);
 
-                        if (pointer.InputSourceParent != null)
+                        if (pointer.InputSourceParent != null && pointerByInputSourceParent.ContainsKey(pointer.InputSourceParent))
```

Comment:
Another optimization could be using `TryGetValue` here, so we don't continually index in the `foreach`

Reply:
Done